crc/scripts/email.py

```diff
@@ -32,14 +32,18 @@ def get_description(self):
 In place of an email address, we accept the string 'associated', in which case we
 look up the users associated with the study who have send_email set to True. 
 The reply_to argument can contain an email address.
-The attachments arguments can contain a doc_code or list of doc_codes.
+
+The attachments arguments can contain a doc_code tuple or list of doc_code tuples.
+A doc_code tuple is a two-item tuple containing a doc_code and optional list of files.
+Normally, we include *all* files for each doc_code. The optional list of files allows 
+a configurator to limit the files we include to only the files in the list. 
 
 Examples:
 email(subject="My Subject", recipients=["dhf8r@virginia.edu", pi.email, 'associated'])
 email(subject="My Subject", recipients="user@example.com", cc='associated', bcc='test_user@example.com)
 email(subject="My Subject", recipients="user@example.com", reply_to="reply_to@example.com")
-email(subject="My Subject", recipients="user@example.com", attachments='Study_App_Doc')
-email(subject="My Subject", recipients="user@example.com", attachments=['Study_App_Doc','Study_Protocol_Document'])
+email(subject="My Subject", recipients="user@example.com", attachments=('Study_App_Doc', []))
```

Comment:
I don't love the tuple, but I can't think of a better alternative.
is the ANYWAY that we can make the tuple optional -- and continue to support this format as well:
email(subject="My Subject", recipients="user@example.com", attachments='Study_App_Doc')
email(subject="My Subject", recipients="user@example.com", attachments=['Study_App_Doc','Study_Protocol_Document'])
Because that is VASTLY easier to understand and use.

Reply:
```
# one single doc code - no filtering
email(subject="My Subject", recipients="user@example.com", attachments='Study_App_Doc')

# multiple doc codes - no filtering
email(subject="My Subject", recipients="user@example.com", attachments=['Study_App_Doc','Study_Protocol_Document'])

# one doc code with filtering
email(subject="My Subject", recipients="user@example.com", attachments=('Study_App_Doc', ['some_file']))

# multiple doc codes with filtering--pass [] if a code doesn't need filtering
email(subject="My Subject", recipients="user@example.com", attachments=[('Study_App_Doc', ['some_file']),('Study_Protocol_Document',[])])

```